stage1_fly/run/main.go

```diff
@@ -110,6 +110,31 @@ func parseFlags() *stage1commontypes.RuntimePod {
 	return &rp
 }
 
+// writeEnv writes the environment to the env file for subsequent rkt enter
+func writeEnv(env []string, envFilePath string) error {
```

Comment:
It looks like something very similat to this logic already exists as `common.WriteEnvFile()`. Can you please try to refactor your PR to use that (augmenting it accordingly if needed)?